package.json

```
@@ -42,16 +43,19 @@
     "graphql-request": "4.1.0",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-hook-form": "7.27.0",
+    "react-hook-form": "^7.43.1",
     "react-redux": "7.2.6",
     "react-router-dom": "6.2.1",
-    "reactjs-localstorage": "1.0.1",
+    "reactjs-localstorage": "^1.0.1",
     "redux-persist": "6.0.0",
     "redux-saga": "1.1.3",
     "rxjs": "7.8.1",
     "subscription-client": "0.9.15",
     "ts-node": "10.5.0",
-    "typed-redux-saga": "1.4.0"
+    "typed-redux-saga": "^1.5.0",
+    "typescript-http-client": "^0.10.4",
+    "web3": "^1.9.0",
+    "web3-utils": "^1.10.0"
```

```suggestion
    "web3-utils": "1.10.0"
```
Please remove carets from package versions, we would like to use static versions if possible